<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>simple Storm</string>
  </property>
  <widget class="QWidget" name="m_mainWindowWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="MainView" name="m_mainView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Coordinates_List"/>
    <addaction name="actionProcess_Raw_Measurement"/>
    <addaction name="actionCreate_Filter"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExtras"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTransformation_Matrix">
   <property name="text">
    <string>Transformation Matrix...</string>
   </property>
  </action>
  <action name="actionExport_Composed_image">
   <property name="text">
    <string>Export Composed image...</string>
   </property>
  </action>
  <action name="actionExport_as_stack_of_images">
   <property name="text">
    <string>Export as stack of images...</string>
   </property>
  </action>
  <action name="actionAuto_detect_beads">
   <property name="text">
    <string>Auto-detect beads</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClear_all">
   <property name="text">
    <string>Discard loaded images (Reset)</string>
   </property>
  </action>
  <action name="actionExport_transformed_coordinates">
   <property name="text">
    <string>Export transformed Coordinates as List...</string>
   </property>
  </action>
  <action name="actionOpen_Coordinates_List">
   <property name="text">
    <string>Open Coordinates List...</string>
   </property>
  </action>
  <action name="actionProcess_Raw_Measurement">
   <property name="text">
    <string>Process Raw Measurement...</string>
   </property>
  </action>
  <action name="actionCreate_Filter">
   <property name="text">
    <string>Create Filter...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainView</class>
   <extends>QWidget</extends>
   <header>mainview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
